src/app.ts

```diff
@@ -47,7 +53,9 @@ nunjucksEnv.addGlobal("OE_CONFIGS", config);
 nunjucksEnv.addGlobal("ERROR_MESSAGES", ErrorMessages);
 nunjucksEnv.addGlobal("COUNTRY_FILTER", countryFilter );
 nunjucksEnv.addGlobal("CREATE_CHANGE_LINK", createChangeLinkConfig);
+nunjucksEnv.addGlobal("CREATE_CHANGE_LINK_WITH_IDs", createChangeLinkWithIds);
 nunjucksEnv.addGlobal("SUMMARY_LIST_LINK", createSummaryListLink);
+nunjucksEnv.addGlobal("IS_REDIS_REMOVAL_ENABLED", isActiveFeature(config.FEATURE_FLAG_ENABLE_TRUSTS_WEB));
```

Comment:
the config object is already passed to the template and the templates can already use the redis feature flag - https://github.com/companieshouse/overseas-entities-web/blob/main/views/beneficial-owner-gov.html#L16
Adding another call to isActiveFeature here will mean a lot of tests will need updating.

Reply:
Just tested this locally with `FEATURE_FLAG_ENABLE_REDIS_REMOVAL` in the view and it's not working. Happy to refactor to use it if it works. Perhaps am using it wrong?

Reply:
try OE_CONFIGS.FEATURE_FLAG_ENABLE_REDIS_REMOVAL

Reply:
Have tested `OE_CONFIGS.FEATURE_FLAG_ENABLE_REDIS_REMOVAL` locally and it works. Will refactor and use that.

Reply:
Now working on the unit tests and it looks like `OE_CONFIGS.FEATURE_FLAG_ENABLE_REDIS_REMOVAL` causes a lot of failures for the `check-your-answers` test suites. Reverting to my earlier changes fixes the issue.

So, we have:

- `FEATURE_FLAG_ENABLE_REDIS_REMOVAL` fails for journey but works for tests
- `OE_CONFIGS.FEATURE_FLAG_ENABLE_REDIS_REMOVAL` works for the journey but fails for the tests
- `IS_REDIS_REMOVAL_ENABLED` works for both the journey and the tests

Will give you a shout so I can show you in a screen-share session when you have a moment. Otherwise, if you don't mind, will revert to what I'd built before, i.e. use `IS_REDIS_REMOVAL_ENABLED` as this works for both.

Cheers.

Reply:
We had that issue previously and solved it with this approach: https://github.com/companieshouse/overseas-entities-web/blob/main/src/controllers/beneficial.owner.type.controller.ts#L30

Reply:
ok, got it

my only issue is that passing it in the controller just means if we have a similar issue on another page, then we'll have to pass in that value in that other controller again. this leads to to fragmentation everywhere. doing it the way i've done it in `app.ts` means we only do it once and we reuse that value in any view where we need it. we avoid the controller-to-controller repetition.